src/natcap/invest/ndr/ndr.py

```diff
@@ -1362,8 +1408,7 @@ def _map_surface_load(
     """Calculate surface load from landcover raster.
 
     Args:
-        modified_load_path (string): path to modified load raster with units
-            of kg/pixel.
```

Comment:
@emilyanndavis I think this unit reference is now outdated since your per hectare PR update.